stacks/java-websphereliberty-gradle/devfile.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
   version: 0.3.1
   displayName: 'WebSphere Liberty Gradle'
   description: Java application Gradle-built stack using the WebSphere Liberty runtime
-  icon: https://raw.githubusercontent.com/OpenLiberty/logos/7fbb132949b9b2589e18c8d5665c1b107028a21d/logomark/svg/OL_logomark.svg
+  icon: https://raw.github.ibm.com/websphere/logos/main/liberty/liberty-was-500-purple.svg?token=AAAJJDABPW7QHG2QWS75L5DCWNSDG
```

Comment:
Can we link the icons from somewhere other than IBM's GHE? If that token ever expires, then the icon won't be available.

Reply:
Hi John - agreed. I'm looking to see if there is another location we can reference other that GHE. Keeping the PR draft until then.

Reply:
@johnmcollier - Links are updated now to a public GH location.